src/containers/AddProjectDetails/AddProjectDetailsForm.style.ts

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export const topNavBgColor = styled.div`
+    background-color: #1f70c1;
+`;
+
+export const leftNavBgColor = styled.div`
+    background-color: #dcdcd9;
```

Comment:
Can we access the color property from the ThemeProvider?

Reply:
will discuss